<launch>
	<arg name="car" default="true"/>
	<arg name="pedestrian" default="false"/>
	<arg name="use_gpu" default="true"/>
	<arg name="gpu_device_id" default="0"/>
	<arg name="score_threshold" default="0.3"/>
	
	<!-- arguments list -->
	
	<arg name="network_definition_file" default="$(env HOME)/darknet/cfg/yolo.cfg"/>
	<arg name="pretrained_model_file" default="$(env HOME)/darknet/data/yolo.weights"/> 

	<arg name="camera_id" default="/" />
	<arg name="image_src" default="/image_raw"/>
	
	<!-- SSD -->
	<node pkg="cv_tracker" name="yolo2_wa" type="yolo2_wa">
		<param name="network_definition_file" type="str" value="$(arg network_definition_file)"/>
		<param name="pretrained_model_file" type="str" value="$(arg pretrained_model_file)"/>
		<param name="use_gpu" type="bool" value="$(arg use_gpu)" />
		<param name="score_threshold" type="double" value="$(arg score_threshold)"/>
		<param name="gpu_device_id" type="int" value="$(arg gpu_device_id)"/>
		<param name="image_raw_node" type="str" value="$(arg camera_id)$(arg image_src)"/>
	</node>
	
</launch>
